src/controllers/applicationsContentController.js

```diff
@@ -0,0 +1,304 @@
+const { renderTemplate } = require('../utils/util');
+const config = require(process.cwd() + '/config');
+const cpToken = require(process.cwd() + '/cpToken');
+const constants = require('../utils/constants');
+const path = require('path');
+const fs = require('fs');
+const axios = require('axios');
+const https = require('https');
+
+const filePrefix = config.pathToContent;
+const certPath = path.join(process.cwd(), config.certificate.path);
+const certPassword = config.certificate.password;
+const controlPlaneUrl = config.controlPlaneUrl;
+const token = cpToken.token;
+
+const httpsAgent = new https.Agent({
+    ca: fs.readFileSync(certPath),  
```

Comment:
Please use a key store validation instead of a single cert to support multiple gateway handshake, something like, const httpsAgent = new https.Agent({
    pfx: fs.readFileSync('/path/to/keystore.pfx'),
    passphrase: 'passphrase',
    rejectUnauthorized: true
});

Reply:
Tried earlier with a approach similar to this but did not work. 

Lets add a separate task for this - https://github.com/wso2/api-manager/issues/3415